big_line.ipynb

```diff
@@ -0,0 +1,109 @@
+eFkAYND3ZnS00ibtYd5Y0c6hzOriPR25LYh4nr4mSkx7Ewp45WtqZTUteHlZMNDijYPj41gw9kilm9Kxs3BGku1Sp7o3TUshO9OFrBySwoDg4U2LyjazEjw4+tj+byxI43sqiaKalv57v7BqC1UDA33EKbz+7PRIVrV6rWJ8nHijiHBfHUsjwA3e/amVfDc1Bi8nBRtxkbwxnahzQ+nCrlneCgRimXP05NjWPjzBd7fn8X6A9kkJfjJK7Kl0+O5/dMTvL07kx9PFTIo1J2pvRRtpsczfe0h3tiRztGsKsK9HOTV+TNTYtiZXM7LW1IprmvF19mWeaPCpDa/ni1i+aYUVCpxBX6PUsjbO8CVO4cE86XiG+vrbIsO4Y+sLyI9lFnFsHAP1t3aT6Y+/T2uPm7sH0CfIFcWfHOWmeuOUFrfRqinA7MHBmKhUrF2bxbv7M4k1NMBHTqWKc0g9N63r2xNZUiYaZxe10fPtzTSyxooqW9jzew+km8TFV/sji7Rwt2cb18fz8fXxZaDGZW8MD1O8m3B6HBW7zLwbd7IMJmmtXhSjAn7r+vjZ8L+uz8/xTt7rmS/nm+rdqRzIKPyquwvqmkRfFMclPTsX74pBWeF/YuVFIe7jdjfP8jNlP0K317fnk6mwrfu2N+ptH9+V9Em2teJ2xXP8B4uduy/Ct8+OpjNT2cKTdlvxrfu2S/4NiTMnckK35YlxTF97WHe3CnYb863XSllvKKsYWP2P2LEfhf7q7N/UMiVfPtKz7dKoY2eb8PCPZnSU7BfqxOtkbvnmx170ypYMjXWwDcz9t87PFS+ND89OYZFP18Q2pjzLSmOOz69Cvsl39I4klVFhLcjt3fD/tJ6wX59g4xHxkWy4Wyx8G1W2K9P/7hzaAjfnRDsHxzqTlN71xXsf3NHOjmVzYL99w3GUm1hxH7hQXypuJ63ZwttIo20CXS3Z0+a6Ong7SRS2B4eG8nr29MU9hd0y/5fzhSx4VwR9a2dfDF3EB6ONvT0c+Z8YR2WagtuHRzEsHAPkzSa9i4tu58cSYS3yBowb1/9V43/sQdklUrlrNPpGv5d39/V3pov5g6k14s7Zf7Ru3sylSstsbHp3S6ifJw4oPQOj1ACW9/tqZe/i5zzVo7+9Z1ohkd4cljJwbSysMDSQsX7Sn7N2Bhvmb8EoupV7wDw4OhwmRMH0NPfRbpd3NAvQFYOg6i61pvAJ0Z6klHWREaZyAsaE+3NF4q5fJSPI2X17ZTVi6Kh8bE+vLkjnY8P5eJiZ0WXRid/1lFR3uxILpenBa72VnJuUKg7Xx/Pl9fQ4V6Ock7fOelVxQi8b6CrnNNXoOu1GWGkjVqlwlptIbUZF2uqTYCbneyKNH+UqTa9/F1kk4/r+/pz3kiboWEecm54hAcZ5U1klAttRkd7KV6NzUR6O1Le0E55Q7vy9/sosM/B2dYSjdagzcgoL7Ynl0lt3B2s5dzAUHe+PJYvjdBNtXEy0aZfkJuc0+e76rvXjYjwlJXLFirR6vgDo7g5nFUlnSf83exkN60HRoZJ03eAnn4uMjZm9vHjrFI5DDA4zIMtSkwNCXMnvayRdKUYbVSUF18cFb6fYZ4OlDe0ySYbY2K8FdiLK7UujY6DGZWoVJAY6cXWS2XytMBYmwEhbnxxNE9qE+HtqFT2Q6S3qTZ9g1zlnKeT6ZpKNIobEDnDem3Gm8VNkLu91Oa+EaEcy65Gh6jaj/dzlnUFMxL8pG46HfQLcpWV65Hejqzdm0V1UzvVTR3kVjfL71/W0Ma8r88AhipyELmipfWt/xEPyP9uDoPg1Fs3J/DgtyJ/PLeqmXlfn5HWmReL6smsaGJomAd5VS2ys9jYGG8+PiS8aCO8HalsbJeF1hPiDHxztbei05hv0V7sTCmXXUa745s+TsO8HOScvvuZcZxKvrmY8s2Y/ZYWFlipVbLhkjn7A9yuzX79Or2+rz9nC2rl2h8U4i5jeFi4B2lljaSVNaJSwehrsN+Yb1djv14bNyNtBoa4yxcP6J5vr2y9kv0+/4j9RnwbF+Njok2gm71k/7xRYZz4J9k/JNzAt6uxP6dSxE1FQzsVkv3e4mX9YA5OtpZotZiwf9vla/PtVSO+XZX9wQb2eyt807PfmG8qVNhaGWnTDd++NWK/sTa9/V2lK891ffxM98VwD/k8MTTMg6yKJplnPSbGm08P55JT2Uy4l4PJmhoX6yMeZg/m4GRjiVZnpI3Cfv2+6OFobRI3xuyP9DFoE+3raKJNfyNt9C8Y+u59r17fS67BPoFufHcyn4rGNj49nMtXR/NlLYJ+PPzdOV6e1VO+jP07xr/sYqFSqZyBp4HTQIhOp1tjNOcEnAPqlP/UotPpRqpUqjhgM6AFVul0uvXX+jv+lerpqCXbuGdEKL38XXhzR5psUwsCjsW1rZQ1tJkI/ff4e/w9/u8OSwsVHo7WeDjY0Nqpkd2cYnydeO2G3oR7OXCpuJ5bPz7BfSNC2XSxhNrmTp6bGsMbO9K5dVCQSXv4f3b8FS4W/1s5/PPpQp765SKbHxnBxgslfHE0z8S5Iq6HM0W1LbKo+O/x9/h7/D18nG0IdnfgZJ7hZUClguv6+PPouEhmrDvMDf0CGBYuOu+V1LcxZ1AgMb7OLNuYzN6Fo+TNxp8Z/w4XiyXAPp1Ot12lUr2uUqkG63Q6fQlyANBTp9O1qVSqSOA65b/fCgzX6XSl3Xy/v2R0aLTyZDjax4nP7h7Az6eLuFxSz0/zhgKwO6Wc+74yAP+neUNltWh9a4fsVQ/wxdyBSj92HRot3PzhMTn3zi19ZM9xnQ5u//SE3AReTIojXjFrB1j08wV5MvLwmAhGGRX2vLk9XQbEzQMCpCE5wJdH8+Tb4KgoL5Ne49sulUmHjRhfJ1Ze11POnc6rlScB7g7WfHiHoR97bmWzfGsD+PGBIbKSuLqpnfnfnJVz3943WPZjb+3QcOdnJ+XcJ3cOkGbtGq2OWz46Lufent0HfzeDNnd+
```

Comment:
That extra space looks a bit weird... let's remove it.

This should look like: 
![image](https://user-images.githubusercontent.com/963242/143091764-8c8a4cd6-c9b5-42be-9e66-01064765450d.png)

but noooo!! :scream: 
```suggestion
    "a_variable = extra_space_function()\n",
```